spec/datadog/appsec/actions_handler_spec.rb

```diff
@@ -0,0 +1,107 @@
+# frozen_string_literal: true
+
+require 'datadog/appsec/spec_helper'
+
+RSpec.describe Datadog::AppSec::ActionsHandler do
+  describe '.handle' do
+    let(:generate_stack_action) do
+      { 'generate_stack' => { 'stack_id' => 'foo' } }
+    end
+
+    let(:generate_schema_action) do
+      { 'generate_schema' => {} }
+    end
+
+    let(:redirect_request_action) do
+      { 'redirect_request' => { 'status_code' => '303', 'location' => 'http://example.com' } }
+    end
+
+    let(:block_request_action) do
+      { 'block_request' => { 'status_code' => '403', 'type' => 'auto' } }
+    end
+
+    it 'calls generate_stack with action parameters' do
+      expect(described_class).to(
+        receive(:generate_stack).with(generate_stack_action['generate_stack']).and_call_original
+      )
```

Comment:
Could this formatting improve the situation? Like that we can keep the main statement in a single line and move details on the next line

```suggestion
      expect(described_class).to receive(:generate_stack)
        .with(generate_stack_action['generate_stack'])
        .and_call_original
```

Reply:
fixed [5a05cab](https://github.com/DataDog/dd-trace-rb/pull/4300/commits/5a05cabb5f260fade144f75e751bc1d3b6e0964f)